locales/en/create-kafka-instance.json

```diff
@@ -1,36 +1,53 @@
 {
+  "all_region_unavailable_helper_text": "All regions in the selected cloud provider are temporarily unavailable. Select another cloud provider or try again later.",
   "availability_zone_selection": "Availability zone selection",
   "availability_zones": "Availability zones",
   "availability_zones_tooltip_message": "At this time, only {{enabledZone}} availability zones are available.",
   "az": {
     "multi": "Multi",
     "single": "Single"
   },
+  "billing": {
+    "field_label": "Billing",
+    "field_popover_body": "TODO",
+    "field_popover_title": "TODO",
+    "marketplace": "Marketplace",
+    "marketplace_aws": "$t(common:cloudProviders.aws) $t(create-kafka-instance:billing.marketplace)",
+    "marketplace_azure": "$t(common:cloudProviders.azure) $t(create-kafka-instance:billing.marketplace)",
+    "marketplace_over_quota": "The selected size exceedes the quota available for the marketplace billing option.",
+    "marketplace_rh": "$t(common:cloudProviders.rh) $t(create-kafka-instance:billing.marketplace)",
+    "prepaid": "Pre-paid",
+    "prepaid_option": "$t(common:cloudProviders.rh) pre-paid",
```

Comment:
This was a typo in my mockups that I tried to update, but I'm sure I missed a few. 
Can you change all the "pre-paid" instances to "prepaid"?

Reply:
done